firefox_ios/views/firefox_ios_feature_usage_events_not_added.view.lkml

```diff
@@ -0,0 +1,545 @@
+COUNTIF(event_category = 'credit_card' AND event_name = 'autofill_prompt_expanded') AS cc_autofill_prompt_expanded,
+COUNTIF(event_category = 'credit_card' AND event_name = 'autofill_prompt_shown') AS cc_autofill_prompt_shown,
+COUNTIF(event_category = 'credit_card' AND event_name = 'autofilled') AS cc_autofilled, --added already
+COUNTIF(event_category = 'credit_card' AND event_name = 'form_detected') AS cc_form_detected, --added already
+COUNTIF(event_category = 'credit_card' AND event_name = 'management_add_tapped') AS cc_management_add_tapped,
+COUNTIF(event_category = 'credit_card' AND event_name = 'management_card_tapped') AS cc_management_card_tapped,
+COUNTIF(event_category = 'credit_card' AND event_name = 'save_prompt_create') AS cc_save_prompt_create,
+COUNTIF(event_category = 'credit_card' AND event_name = 'save_prompt_shown') AS cc_save_prompt_shown,
+COUNTIF(event_category = 'credit_card' AND event_name = 'save_prompt_update') AS cc_save_prompt_update,
+
+/*History*/
+COUNTIF(event_category = 'history' AND event_name = 'opened') AS history_opened, --added already
+COUNTIF(event_category = 'history' AND event_name = 'opened_item') AS history_opened_item,
+
+/*Privacy*/
+COUNTIF(event_category = 'tabs_tray' AND event_name = 'private_browsing_icon_tapped') AS private_browsing_icon_tapped, --added already
+COUNTIF(event_category = 'tabs_tray' AND event_name = 'private_browsing_icon_tapped' AND extra.value = 'add') AS private_browsing_button_tapped_add,
+COUNTIF(event_category = 'tabs_tray' AND event_name = 'private_browsing_icon_tapped' AND extra.value = 'close_all_tabs') AS private_browsing_button_tapped_close_all_tabs,
+COUNTIF(event_category = 'tabs_tray' AND event_name = 'private_browsing_icon_tapped' AND extra.value = 'done') AS private_browsing_button_tapped_done,
+
+/*Notification*/
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'status' AND extra.value = 'authorized') AS notification_status_authorized,
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'status' AND extra.value = 'notDetermined') AS notification_status_notDetermined,
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'status' AND extra.value = 'denied') AS notification_status_denied,
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'alert_setting' AND extra.value = 'notSupported') AS notification_alert_setting_not_supported,
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'alert_setting' AND extra.value = 'disabled') AS notification_alert_setting_disabled,
+COUNTIF(event_category = 'app' AND event_name = 'notification_permission' AND extra.key = 'alert_setting' AND extra.value = 'enabled') AS notification_alert_setting_enabled
+FROM `mozdata.firefox_ios.events_unnested`
+LEFT JOIN
+    UNNEST(event_extra) AS extra
```

Comment:
did you ensure that all the events that you are counting without the extra condition are not being double counted for whatever reason if they happen to have multiple extra field?